terraform/gcp/security/gcp-sqlserver-no-public-ip.yaml

```diff
@@ -0,0 +1,26 @@
+rules:
+- id: gcp-sqlserver-no-public-ip
+  patterns:
+    - pattern: resource
+    - pattern-inside: |
+        resource "google_sql_database_instance" "..." {
+        ...
+        ip_configuration {
+          ...
+          ipv4_enabled = true

```

Comment:
Is this really all that's required to give the DB instance a public IP? 😮  Crazy

Reply:
According to https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/sql_database_instance#ipv4_enabled, yes.